TestApps/TestAppSwiftUI/ContentView.swift

```diff
@@ -24,7 +26,10 @@ struct ContentView: View {
 
     var body: some View {
```

Comment:
Can we at least display the test app name on top of the other controls? version would be helpful too